src/pages/index.js

```diff
@@ -0,0 +1,290 @@
+const profileAvatar = document.querySelector(".profile__avatar");
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const editProfileModal = document.querySelector("#edit-profile-modal");
+const modalEditProfileCloseButton =
+  editProfileModal.querySelector(".modal__close-btn");
+
+//form elements
+const editModal = document.querySelector("#edit-profile-modal");
+const profileName = document.querySelector(".profile__name");
+const editFormElement = document.querySelector(".modal__form");
+const editModalNameInput = document.querySelector("#profile-name-input");
+const profileDescription = document.querySelector(".profile__description");
+const editModalDescriptionInput = document.querySelector(
+  "#profile-description-input"
+);
+//deleteCard modal
+const deleteCardModal = document.querySelector("#deleteCard-modal");
+const deleteCardButton = deleteCardModal.querySelector(
+  ".modal__deleteCard-btn"
+);
+const cancelDeleteCardButton = deleteCardModal.querySelector(
+  ".modal__cancelDeleteCard-btn"
+);
+const deleteCardForm = deleteCardModal.querySelector(".modal__form");
+//avatarModal
+const avatarModal = document.querySelector("#edit-avatar-modal");
+const avatarModelButton = document.querySelector(".profile__avatar-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+const avatarForm = avatarModal.querySelector(".modal__form");
+//Card modal
+const cardModal = document.querySelector("#add-card-modal");
+const cardModalCloseBtns = cardModal.querySelector(".modal__close-btn");
+const cardModelButton = document.querySelector(".profile__add-btn");
+const cardForm = cardModal.querySelector(".modal__form");
+
+//card modal elements
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
+
+//card related elements
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+//selectmodal
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageElement = document.querySelector(".modal__image");
+const previewModalCaptionElement = document.querySelector(".modal__caption");
+const modalImageCloseButton = document.querySelector(
+  ".modal__close-btn-preview"
+);
+// functions
+function handleAddCardFormSubmit(evt) {
+  evt.preventDefault();
+  const inputValues = { name: cardNameInput.value, link: cardLinkInput.value };
+  const cardElement = getCardElement(inputValues);
+  const submitbtn = evt.submitter;
+  setButtonText(submitbtn, true, "Save", "Saving...");
+  api
+    .editCardsInfo({ name: cardNameInput.value, link: cardLinkInput.value })
+    .then(() => {
+      cardsList.prepend(cardElement);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(submitbtn, false, "Save", "Saving...");
+      closeModal(cardModal);
+      evt.target.reset();
+      window.location.reload();
```

Comment:
Please, delete ` window.location.reload();`


* When I add a new card, the page **reloads**, but it should not happen on submission